tutorials/run-spring-petclinic-on-app-engine-flex-cloudsql.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Run Spring Pet Clinic with CloudSQL on Google App Engine Flexible Environment
+description: Learn how to deploy Spring Boot Pet Clinic application to Google App Engine flexible environment and use CloudSQL.
+author: jishaa
+tags: App Engine Flex, CloudSQL, Spring Boot
```

Comment:
s/App Engine Flex/App Engine/ (for consistency with other tutorials, and because we don't call it "App Engine Flex" externally)